lib/fun_with_flags/ui/templates/rows/_percentage.html.eex

```diff
@@ -6,18 +6,16 @@
   end
 %>
 <div id="percentage_gate" class="container fwf-I-hate-grids">
-  <div class="row no-gutters">
-    <div class="col-lg-6 col-md-6 col-sm-5 col-4 text-left">
+  <div class="row no-gutters d-flex flex-nowrap" style="gap: 8px;">
+    <div class="flex-grow-1 w-100">
       <span class="badge badge-default">% of <%= gate_type %></span>
       -
       <%= html_status_for @gate.enabled %> for <%= Utils.as_percentage(@gate.for) %>% of the <%= gate_type %>
-    </div>
 
-    <div class="col-lg-5 col-md-5 col-sm-5 col-5 text-left">
```

Comment:
I'd try to keep this `div` around the "raw value" bit of info, because I've notices that without it the "raw value" text gets rendered directly as a continuation of the previous part of the row. The result doesn't make much sense in English, for example:

> Enabled for 30.0% of the actors raw value: 0.3